### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by OlegXXL**
pause works in the game on the "TAB" button
also check the loading between scenes, and check the main menu

### CODE REVIEW

**Comment by loburyaka** [comment]
Everything is OK.

**Review by loburyaka — APPROVED** [review]
Everything is OK.

**Review by DimaDiD — COMMENTED** [review]
NICE WORKKKK

File: Assets/Scripts/ChooseMap.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    void Start()
+    {
+        levelComplete = PlayerPrefs.GetInt("levelComplete");
+        map1.interactable = true;
+        map2.interactable = false;
+        map3.interactable = false;
+
+        switch (levelComplete)
+        {
+            case 1:
```
Comment (ppolyanskiyy) [inline Assets/Scripts/ChooseMap.cs:25]: Please, add enum support

**Review by SwaTiK4931 — APPROVED** [review]

**Comment by OlegXXL** [comment]
> pause works in the game on the "TAB" button
> also check the loading between scenes, and check the main menu

**Pull request merged**